runtime/v2/manager.go

```diff
@@ -511,3 +514,34 @@ func (m *TaskManager) Delete(ctx context.Context, taskID string) (*runtime.Exit,
 
 	return exit, nil
 }
+
+func (m *TaskManager) RuntimeInfo(ctx context.Context, runtimeName string, runtimeOptions interface{}) (*task.RuntimeInfo, error) {
+	runtimePath, err := m.manager.resolveRuntimePath(runtimeName)
+	if err != nil {
+		return nil, fmt.Errorf("failed to resolve runtime path: %w", err)
+	}
+	var optsB []byte
+	if runtimeOptions != nil {
+		optsPBAny, err := protobuf.MarshalAnyToProto(runtimeOptions)
+		if err != nil {
+			return nil, fmt.Errorf("failed to marshal %T: %w", runtimeOptions, err)
+		}
+		optsB, err = proto.Marshal(optsPBAny)
+		if err != nil {
+			return nil, fmt.Errorf("failed to marshal %T: %w", optsPBAny, err)
+		}
+	}
+	var stderr bytes.Buffer
+	cmd := exec.CommandContext(ctx, runtimePath, "-info")
+	cmd.Stdin = bytes.NewReader(optsB)
+	cmd.Stderr = &stderr
+	stdout, err := cmd.Output()
+	if err != nil {
+		return nil, fmt.Errorf("failed to run %v: %w (stderr: %q)", cmd.Args, err, stderr.String())
+	}
+	var info task.RuntimeInfo
+	if err = proto.Unmarshal(stdout, &info); err != nil {
+		return nil, fmt.Errorf("failed to unmarshal stdout from %v into %T: %w", cmd.Args, &info, err)
```

Comment:
`fmt.Errorf("failed to unmarshal stdout from %s into %T: %w", string(stdout), &info, err)` ?

Reply:
Do stdout and stderr have to be part of the error message? They are uncontrollable in size

Reply:
stdout is a binary, so `string(stdout)` doesn't make a sense here